app/components-react/highlighter/EducationCarousel.m.less

```diff
@@ -0,0 +1,186 @@
+@paddingLeft: 24px; // was 24
+@paddingTop: 20px;
+@paddingRight: 24px;
+@paddingBottom: 20px;
+
+:global {
```

Comment:
so we don't want to have global unscoped changes in our modular css files-- when they get transpiled they'll insert global rules through the whole app. we should have some sort of `.carousel-container` to wrap these in